ddtrace/internal/injection.py

```diff
@@ -66,13 +78,15 @@ def _inject_hook(code, hook, lineno, arg):
         )
 
 
-_INJECT_HOOK_OPCODES = (
-    ["LOAD_CONST", "LOAD_CONST", "CALL_FUNCTION", "POP_TOP"]
-    if sys.version_info < (3, 11)
-    else ["PUSH_NULL", "LOAD_CONST", "LOAD_CONST", "PRECALL", "CALL", "POP_TOP"]
-)
-_INJECT_HOOK_OPCODE_POS = 0 if sys.version_info < (3, 11) else 1
-_INJECT_ARG_OPCODE_POS = 1 if sys.version_info < (3, 11) else 2
+# Default to Python 3.11 opcodes
+_INJECT_HOOK_OPCODES = ["PUSH_NULL", "LOAD_CONST", "LOAD_CONST", "PRECALL", "CALL", "POP_TOP"]
+if PY < (3, 11):
+    _INJECT_HOOK_OPCODES = ["LOAD_CONST", "LOAD_CONST", "CALL_FUNCTION", "POP_TOP"]
+elif PY >= (3, 12):
+    _INJECT_HOOK_OPCODES = ["PUSH_NULL", "LOAD_CONST", "LOAD_CONST", "CALL", "POP_TOP"]
```

Comment:
Why is load_const listed twice?

Reply:
we have multiple arguments to load, if you look at the example in `injection.py` it might help make it more clear. That code uses `Instr()` which includes the opcode and the arguments, this definition is the opcodes only, and we'll combine with the arguments later.

https://github.com/DataDog/dd-trace-py/pull/5875/files#diff-daea50be73be7ab1aff2a5dde30e9c185d3ea7f870a1a27806a91a2e4a75c5ecR58-R65

Reply:
Ah, got it, thank you!